doc/release-notes.md

```diff
@@ -65,13 +80,43 @@ are consistent, so that extended keys from one can be used with the other. This
 future compatibility with projects such as HWI, which could enable Dogecoin Core to use hardware
 wallets.
 
+Version UI
+----------
+
+Dogecoin version is now displayed in the bottom-left of the window, rather than in the
+Dogecoin logo in the wallet window. This means it is updated automatically, rather than
+requiring manual work editing the image.
+
+createauxblock/submitauxblock
+-----------------------------
+
+The `createauxblock` and `submitauxblock` commands have been introduced, mimicking
+the same commands from Namecoin 0.17+. These progress towards splitting the consensus
+layer from the wallet (`getauxblock` generates an address to mine to, while the
+new `createauxblock` command takes in an address to mine to, and therefore does not
+require the wallet). It also enables mining pools to use multiple wallet addresses
+if desired.
+
+By default `createauxblock` provides difficulty target in a field called `target`,
+however this can be tweaked for Namecoin-like API ("_target") by
+setting the `-rpcnamecoinapi` startup argument.
+
 Minor Changes
 =============
 
 * Fix compilation on FreeBSD, which was failing to compile the Scrypt code.
 * Refresh FreeBSD docs, see `doc/build-freebsd.md`.
 * Update to OpenSSL 1.0.2u.
 * Refresh translation files to simplify volunteer contributions to translations.
+* Remove export of glibc 2.17 secure_getenv to fix glibc minimum required.
+* Add xkbcommon 0.8.4 to fix keyboard compatibility issues.
+* Address compatibility issues with GCC-7 and glibc-2.27.
+* Remove legacy patches which are no longer needed due to test environments being updated to more recent Ubuntu releases.
+* Security harden systemd unit file `contrib/init/dogecoind.local.service`.
+* Make Freetype library version independent from build system libaries.
+* Update Univalue library to 1.0.4.
+* Correct block download timeout for regtest, where it was too low and causing issues with tests.
+* Experimental build for SHA algorithms with AVX2 support, to improve SHA performance.
 
 Credits
```

Comment:
you're missing some authors in co-author lines in squashed commits:

```
* AbcSxyZ
* Bertrand Jacquin
* Carl Dong
* cg
* chromatic
* Chun Kuan Lee
* Cory Fields
* Dakoda Greaves
* Daksh Sharma
* Dan Raviv
* dogespacewizard
* Ed Tubbs
* Elvis Begović
* fanquake
* Hennadii Stepanov
* KabDeveloper
* leezhen
* Micael Malta
* Michi Lumin
* Patrick Lodder
* Ross Nicoll
* Ryan Crosby
* Suhas Daftuar
* W. J. van der Laan
```

Reply:
Done, thanks